evaluator/evaluator_test.go

```diff
@@ -743,33 +683,25 @@ func (s *testEvaluatorSuite) TestLike(c *C) {
 		match := doMatch(v.input, patChars, patTypes)
 		c.Assert(match, Equals, v.match, Commentf("%v", v))
 	}
-	cases := []testCase{
-		{
-			exprStr:   "'a' LIKE ''",
-			resultStr: "0",
-		},
-		{
-			exprStr:   "'a' LIKE 'a'",
-			resultStr: "1",
-		},
-		{
-			exprStr:   "'a' LIKE 'b'",
-			resultStr: "0",
-		},
-		{
-			exprStr:   "'aA' LIKE 'Aa'",
-			resultStr: "1",
-		},
-		{
-			exprStr:   "'aAb' LIKE 'Aa%'",
-			resultStr: "1",
-		},
-		{
-			exprStr:   "'aAb' LIKE 'Aa_'",
-			resultStr: "1",
-		},
-	}
-	s.runTests(c, cases)
+	f := Funcs[ast.Like]
```

Comment:
why you prefer repeat write down function call some many times instead of make a testcase table?